Misc/NEWS.d/next/Library/2019-10-16-08-08-14.bpo-38490.QbDXEF.rst

```diff
@@ -0,0 +1 @@
+Covariance, Pearson's correlation, and simple linear regression functionality was added to statistics module. Patch by Tymoteusz Wołodźko.
```

Comment:
Did you mean "functionality" or "functions"? I am good with either choice, but "functions" might be slightly less unusual.

Reply:
@twolodzko has already applied my suggested changes to that, which include replacing "functionality" with "functions".